extensions/Worker.Extensions.ServiceBus/src/Worker.Extensions.ServiceBus.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Azure Service Bus extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>5.20.1</VersionPrefix>
+    <VersionPrefix>5.20.2</VersionPrefix>
```

Comment:
I think we should do a minor version bump for this package as the `Azure.Messaging.ServiceBus` dependency was updated with a minor version bump

Reply:
Good point. Done